.github/workflows/jest-coverage-changes.yml

```diff
@@ -0,0 +1,31 @@
+name: Jest Coverage - changed files
+
+on:
+  pull_request:
+    branches: develop
```

Comment:
`develop`, `main` and `release/*` branches would good to include here.

Reply:
Discussed with team and as this is only `jest coverage report` that will only report the coverage % which does not seems to be useful for other than develop.

note: JEST test case check running all test cases will always run against every branched (main, release, develop) (that one is a separate check)